src/annotator/anchoring/test/pdf-test.js

```diff
@@ -724,4 +725,35 @@ describe('annotator/anchoring/pdf', () => {
       });
     });
   });
+
+  describe('isTextLayerRenderingDone', () => {
+    [true, false].forEach(renderingDone => {
+      it('returns renderingDone if present', () => {
+        assert.equal(
+          isTextLayerRenderingDone({ renderingDone }),
+          renderingDone,
+        );
+      });
+    });
+
+    it('returns false if neither renderingDone nor the div are set', () => {
+      assert.isFalse(isTextLayerRenderingDone({}));
+    });
+
+    [
+      { element: null, expectedResult: false },
+      { element: {}, expectedResult: true },
+    ].forEach(({ element, expectedResult }) => {
+      it('returns true if the div contains an endOfContent element', () => {
+        assert.equal(
+          isTextLayerRenderingDone({
+            div: {
+              querySelector: () => element,
```

Comment:
This could use an actual DOM element for `div` to decouple it from the method used to test for the class.